Doc/library/functools.rst

```diff
@@ -481,6 +481,9 @@ The :mod:`functools` module defines the following functions:
    See :func:`itertools.accumulate` for an iterator that yields all intermediate
    values.
 
+   .. versionchanged:: 3.14
```

Comment:
We have this now :)
See [here](https://discuss.python.org/t/versionadded-next/65280)
```suggestion
   .. versionchanged:: next
```